assets/app/views/project.html

```diff
@@ -149,9 +149,11 @@ <h2 ng-if="!displayRouteByService[serviceId]">
                             and {{numRemaining}} {{numRemaining == 1 ? "other" : "others"}}
                           </span>
                         </span>
+                        <div class="spacer">&nbsp;</div>
```

Comment:
This is less than ideal from a markup semantics perspective.  Ideally, since you are adding an optional second link here, you'd use a ul.list-inline to house the Create Route and service links and adjust the corresponding CSS accordingly.  To get the pipe divider, you could do something like:

.relevant-class li + li {
  margin-left: 20px;
  &:before {
    background-color: @gray;
    content: '';
    height: 100%;
    left: -10px;
    position: absolute;
    top: 0;
    width: 1px; 
  }
}

This CSS will only put margin and a pipe before a link if it is proceeded by a link.

Note:  this is not exact CSS!  I'm just spitballing in a comment.  :)